### ISSUE TITLE
How to use .find_min_global() with other deep learning framework?

### ISSUE THREAD

**Issue by dbsxdbsx** [davisking/dlib#1052]
I am curious of the  .find_min_global() function that could find near optimal hyper parameter automatically.

Here I use some other framework like Tensorflow, Mxnet to do machine learning with neural network. I hope the .find_min_global() could tell me what kind of learning rate to use to minize error.  

Is it possible ,or even an example?

**Comment by davisking** [comment]
Yes, it's possible.  The example programs show how to use this routine.  Read the example.  It's very short and simple. http://dlib.net/global_optimization.py.html
